.github/actions/setup-sentry/action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
     description: 'Mode to bring up by new devservices'
     required: false
     default: 'default'
+  toggle:
+    description: 'Dependency to toggle off '
```

Comment:
shouldn't this be `dependency to toggle to local` to be explicit?

and then `devservices toggle foo local`